bin/spring

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
```

Comment:
What is Spring?

Reply:
Spring is a Rails application preloader. It speeds up development by keeping your application running in the background so you don't need to boot it every time you run a test, rake task or migration. According to its github page